doc/source/conf.py

```diff
@@ -43,7 +43,7 @@
 
 # General information about the project.
 project = u'AWS CLI Command Reference'
-copyright = u'2018, Amazon Web Services'
```

Comment:
Lets remove this update.

Reply:
Undone with https://github.com/aws/aws-cli/pull/5141/commits/a02823f04ed982e5ddc3d8111346163085eecb04